var/spack/repos/builtin/packages/redland-bindings/package.py

```diff
@@ -0,0 +1,32 @@
+# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class RedlandBindings(AutotoolsPackage):
+    """Redland Language Bindings for language APIs to Redland"""
+
+    homepage = "http://librdf.org/"
+    url      = "http://download.librdf.org/source/redland-bindings-1.0.17.1.tar.gz"
+
+    version('1.0.17.1', sha256='ff72b587ab55f09daf81799cb3f9d263708fad5df7a5458f0c28566a2563b7f5')
+    version('1.0.16.1', sha256='065037ef61e9b78f642e75b9c2a42700eb1a87d903f2f9963d86591c7d916826')
+    version('1.0.14.1', sha256='a8cc365fccf292c56d53341ecae57fe8727e5002e048ca25f6251b5e595aec40')
+
+    depends_on('swig')
```

Comment:
@adamjstewart Should this be:
```suggestion
    depends_on('swig', type='build')
```
? I never used `swig` and don't know exactly how it works. I would find it weird though if a tool to automatically create wrappers becomes a link/run dependency. Looking at other packages doesn't help, since there are a lot of different dep types for `swig`.

Reply:
It's usually `type='build'` in my experience, the other packages are probably wrong.

Reply:
I also changed dependency type of swig in `lib64 check, swig type` commit.